pallets/schema/src/mock.rs

```diff
@@ -72,107 +75,98 @@ pub mod runtime {
-			Balances: pallet_balances::{Pallet, Call, Storage, Event<T>},
+			Schema: pallet_schema::{Pallet, Call, Storage, Event<T>},
+			MockOrigin: mock_origin::{Pallet, Origin<T>},
 		}
 	);
 
 	parameter_types! {
 		pub const SS58Prefix: u8 = 29;
+		pub const BlockHashCount: u64 = 250;
 	}
 
 	impl frame_system::Config for Test {
 		type BaseCallFilter = frame_support::traits::Everything;
 		type BlockWeights = ();
 		type BlockLength = ();
+		type DbWeight = ();
 		type RuntimeOrigin = RuntimeOrigin;
 		type RuntimeCall = RuntimeCall;
-		type Index = u64;
-		type BlockNumber = u64;
+		type Index = Index;
+		type BlockNumber = BlockNumber;
 		type Hash = Hash;
 		type Hashing = BlakeTwo256;
 		type AccountId = AccountId;
 		type Lookup = IdentityLookup<Self::AccountId>;
 		type Header = Header;
-		type RuntimeEvent = ();
-		type BlockHashCount = ConstU64<250>;
-		type DbWeight = RocksDbWeight;
+		type RuntimeEvent = RuntimeEvent;
+		type BlockHashCount = BlockHashCount;
 		type Version = ();
 		type PalletInfo = PalletInfo;
-		type AccountData = pallet_balances::AccountData<Balance>;
+		type AccountData = ();
 		type OnNewAccount = ();
 		type OnKilledAccount = ();
 		type SystemWeightInfo = ();
 		type SS58Prefix = SS58Prefix;
 		type OnSetCode = ();
-		type MaxConsumers = ConstU32<16>;
+		type MaxConsumers = frame_support::traits::ConstU32<16>;
 	}
 
 	parameter_types! {
-		pub const ExistentialDeposit: Balance = 500;
-	}
-
-	impl pallet_balances::Config for Test {
-		type Balance = Balance;
-		type DustRemoval = ();
-		type RuntimeEvent = ();
-		type ExistentialDeposit = ExistentialDeposit;
-		type AccountStore = System;
-		type WeightInfo = ();
-		type MaxLocks = ConstU32<50>;
-		type MaxReserves = ConstU32<50>;
-		type ReserveIdentifier = [u8; 8];
-	}
-
-	parameter_types! {
-		pub const Fee: Balance = 500;
 		pub const MaxSignatureByteLength: u16 = 64;
 		pub const MaxEncodedMetaLength: u32 = 5_000;
+		pub const MaxEncodedSchemaLength:u32 = 5_000;
 	}
-
-	impl Config for Test {
-		type Signature = (CreatorId, Vec<u8>);
-		type CreatorSignatureVerification = EqualVerify<Self::SchemaCreatorId, Vec<u8>>;
-		type SchemaCreatorId = CreatorId;
-		type EnsureOrigin = frame_system::EnsureSigned<Self::AccountId>;
-		type RuntimeEvent = ();
-		type Currency = Balances;
-		type Fee = Fee;
-		type MaxSignatureByteLength = MaxSignatureByteLength;
-		type MaxEncodedMetaLength = MaxEncodedMetaLength;
-		type FeeCollector = ();
+	pub(crate) type TestSchemaCreator = SubjectId;
+	pub(crate) type TestSchemaPayer = AccountId;
+	pub(crate) type TestOwnerOrigin =
+		mock_origin::EnsureDoubleOrigin<TestSchemaPayer, TestSchemaCreator>;
+	pub(crate) type TestOriginSuccess =
+		mock_origin::DoubleOrigin<TestSchemaPayer, TestSchemaCreator>;
+
+	impl pallet_schema::Config for Test {
+		type EnsureOrigin = TestOwnerOrigin;
+		type OriginSuccess = TestOriginSuccess;
+		type RuntimeEvent = RuntimeEvent;
+		type SchemaCreatorId = AccountId32;
```

Comment:
This assignment should be SubjectId, not AccountId32. This is the reason for origin error